bitnami/superset/templates/flower/pdb.yaml

```
@@ -0,0 +1,30 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and .Values.flower.enabled .Values.flower.pdb.create }}
+apiVersion: {{ include "common.capabilities.policy.apiVersion" . }}
+kind: PodDisruptionBudget
+metadata:
+  name: {{ printf "%s-flower" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+    app.kubernetes.io/component: celery-flower
+    app.kubernetes.io/part-of: superset
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if .Values.flower.pdb.minAvailable }}
+  minAvailable: {{ .Values.flower.pdb.minAvailable }}
+  {{- end  }}
+  {{- if or .Values.flower.pdb.maxUnavailable ( not .Values.flower.pdb.minAvailable ) }}
+  maxUnavailable: {{ .Values.flower.pdb.maxUnavailable }}
```

You missed to update other PDBs:
```suggestion
  maxUnavailable: {{ .Values.flower.pdb.maxUnavailable | default 1 }}
```